docs/medical-api/more-info/sandbox.mdx

```diff
@@ -1,14 +1,323 @@
 ---
 title: "Sandbox Mode"
-description: "Build your application using development data"
+description: "Build your application using example clinical data"
 ---
 
-Metriport offers a Sandbox environment so you can develop your medical applications more easily. This will allow you to try out
-the Medical API with some development/dummy data.
+# Overview
 
-After you get access to the Medical API, you can go to the Developers page and toggle the switch under the Sandbox section.
+Metriport offers a Sandbox environment so you can develop your medical applications more easily. This environment
+allows you to try out the Medical API with some example de-identified clinical data (see section below for more details).
 
-Note that you’ll have to generate a separate Sandbox Secret Key.
+After you get access to the Medical API, you can go to the Developers page and toggle the switch under the Sandbox section to
+activate Sandbox mode.
+
+## Example Clinical Data
+
+The Sandbox envirnmont will give you access to comprehensive example clinical data for several
+test patients - this data will be representative of what you can expect to see in production:
+
+- C-CDA documents in XML format.
+- FHIR resources in JSON format.
+- Various de-identified unstructured clinical documents in PDF format.
+- Various de-identified images in formats like TIFF and JPEG.
+
+To access this data, you just need to create Patients that match some pre-defined
+demographics.
+
+<Warning>
+  In Sandbox mode, any Patient you create with a first name that matches one of the pre-defined
```

Comment:
Not sure if it work under `warning`, but I suggest making `first name` bold, so it jumps to the eye while you scan the text.